index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta charset="utf-8">
+    <title>Devon's Portfolio</title>
+    <link rel="stylesheet" href="vendor/styles/normalize.css">
+    <link rel="stylesheet" href="vendor/icons.css">
+    <link rel="stylesheet" href="styles/base.css">
+    <link rel="stylesheet" href="styles/layout.css">
+    <link rel="stylesheet" href="styles/modules.css">
+
+    <script id="portfolio-template" type="text/x-handlebars-template">
+      <header>
+        <h4><a class="project-name" href="{{url}}">{{name}}</a></h4>
+      </header>
+      <section class="project-info">{{{info}}}</section>
+    </script>
+  </head>
+  <body>
+    <main>
+      <header>
+        <nav id="site-nav">
+            <div class="icon-menu"></div>
+            <ul>
+              <li class="site-tab" data-content="home"><a href="#" class="icon-home3"></a>Home</li>
+              <li class="site-tab" data-content="about"><a href="#" class="icon-power"></a>About Me</li>
+              <li class="site-tab" data-content="portfolio"><a href="#" class=
+              <li class=""><a href="https://github.com/devonhackley" class="icon-github"></a>Github</li>
+              <li class=""><a href="#" class="icon-twitter3">Twitter</a></li>
```

Comment:
Not sure if you have a twitter to link to this but would be a simple add. Just follow same guideline as github on the line above.